### PR TITLE
Fix cached looker font sizes

### PR DESCRIPTION
**Pull request by benjaminpkane**
## What changes are proposed in this pull request?

Fixes cached looker font size when resizing the grid

### Before
https://github.com/user-attachments/assets/bb27c924-059d-4ee6-b60a-afc9250ca74c

### After
https://github.com/user-attachments/assets/db1d8ea6-4689-4eec-8567-be7db128ee67

### What areas of FiftyOne does this PR affect?

-   [x] App: FiftyOne application changes
-   [ ] Build: Build and test infrastructure changes
-   [ ] Core: Core `fiftyone` Python library changes
-   [ ] Documentation: FiftyOne documentation changes
-   [ ] Other

## Summary by CodeRabbit

- **New Features**
	- Enhanced `Grid` component with improved error handling and font size management for `looker` instances.
	- Updated `attach` method to accept an optional font size parameter for better rendering control.

- **Bug Fixes**
	- Refined error messages for better clarity regarding data integrity issues.
	- Improved cleanup functions to prevent memory leaks related to event listeners.

- **Style**
	- Changed import statement for React in `ImageContainerHeader` to a named import without affecting functionality.

### CODE REVIEW

**Review by tom-vx51 — DISMISSED** [review]

**Review by sashankaryal — APPROVED** [review]
lgtm

**Review by sashankaryal — CHANGES_REQUESTED** [review]
e2e is failing, revoking approval

**Comment by benjaminpkane** [comment]
Trying to debug this. The failing tests are passing locally 🤔

**Comment by brimoor** [comment]
This works perfectly; can we deal with E2E in a follow-up?

**Comment by benjaminpkane** [comment]
Perhaps. I need a couple hours. There may be some unwanted side effects

File: app/packages/looker/src/lookers/abstract.ts [hunk 1]
```diff
@@ -491,6 +496,7 @@ export abstract class AbstractLooker<
     this.updater({
       windowBBox: dimensions ? [0, 0, ...dimensions] : getElementBBox(element),
       disabled: false,
+      options: { fontSize },
```
[the remark this answers is not in the document]
Comment (benjaminpkane) [reply app/packages/looker/src/lookers/abstract.ts:499]: Handled by looker's internal updater

**Review by sashankaryal — APPROVED** [review]
lgtm!

**Pull request merged**